specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -7189,6 +7485,17 @@
           "description": "The data pending at source virtual machine in MB.",
           "type": "number"
         },
+        "diskState": {
```

Comment:
this should be modeled as enum

Reply:
As far as I know using enums is not considered a good practice. Are you are referring allowed values construct?

Reply:
why is enums not a good practice? you can set modelasString:true and that will not be a breaking change when you add new values.